pachyderm/templates/postgresql/statefulset.yaml

```diff
@@ -37,16 +37,17 @@ spec:
         ports:
         - containerPort: 5432
           name: client-port
-        resources:
-          requests:
-            cpu: "1"
-            memory: 2G
+        {{- if .Values.postgresql.resources }}
```

Comment:
Probably a good idea to do `{{- if and .Values.postgresql .Values.postgresql.resources }}`

Reply:
Normally yes, but in this case, we initialize postgresql in values always, so should be good.